config/yaml_test.go

```diff
@@ -56,6 +56,39 @@ func TestYamlSimple(t *testing.T) {
 	assert.Equal(t, ":28941", c.AsString())
 }
 
+func TestYAMLEnvInterpolation(t *testing.T) {
+	t.Parallel()
+	defer env.Override(t, "OWNER_EMAIL", "hello@there.yasss")()
+
+	cfg := []byte(`
+name: some name here
+owner: ${OWNER_EMAIL}
+module:
+  fake:
+    number: ${FAKE_NUMBER:321}`)
```

Comment:
Actually, I'm not sure if it's a good idea anymore :) What if empty string is the default? 🤔

What then, force `${FOO:""}`?